pallets/omnipool-liquidity-mining/src/lib.rs

```diff
@@ -355,11 +350,11 @@ pub mod pallet {
 			owner: T::AccountId,
 			yield_per_period: Perquintill,
 			min_deposit: Balance,
-			lrna_price_adjustment: FixedU128,
 		) -> DispatchResult {
 			<T as pallet::Config>::CreateOrigin::ensure_origin(origin)?;
 
-			let (id, max_reward_per_period) = T::LiquidityMiningHandler::create_global_farm(
+			//NOTE: Oracle is used as `price_adjustment` provider.
+			let (id, max_reward_per_period) = T::LiquidityMiningHandler::create_global_farm_without_price_adjustment(
```

Comment:
this is bit misleading to me. Comment says that price adjustment is provided by oracle, but then method "create_without_price_adjustment" is called. 

I have quite hard time undestanding at what point the oracle is queried

Reply:
This pallet is built on top of warehouse LM, oracle for `price_adjustment` is plugged into warehouse LM, warehouse LM query oracle for price adjustment. 
Comment here is just so when someone look at it after few months, he knows why we are creating global farm without price adjustment param.

Reply:
@martinfridrich In the liquidity mining pallet, I could not find the implementation of using price oracle for price adjustment. The trait `` sets `price_adjustment` to 1:  
https://github.com/galacticcouncil/warehouse/blob/0047e9ceff47b2a058ae9ecc25da96d1e827a26a/liquidity-mining/src/lib.rs#L1771

Would you advise where can I find the implementation of using oracle price?

Furthermore, is it expected to use an oracle price derived from Omnipool in Omnipool Liquidity Mining? (Assuming in future case, there are data source from other pools). If so, should the price_adjustment be configured by Omnipool Liquidity Mining?